src/util/env.ts

```diff
@@ -1,5 +1,6 @@
 import { isCI as libIsCI } from "ci-info";
 let VERBOSE = false;
+let CI = false;
```

Comment:
```suggestion
let CI = libIsCI;
```

So we won't need to combine two variables in `isCI` function. BTW, this is one is a minor cosmetic suggestion. We don't need to apply it.